modules/apps/calendar/calendar-service/src/main/java/com/liferay/calendar/internal/upgrade/v4_2_1/CalendarBookingUpgradeProcess.java

```diff
@@ -35,14 +35,14 @@ protected void doUpgrade() throws Exception {
 				connection.prepareStatement(
 					SQLTransformer.transform(
 						StringBundler.concat(
-							"select calendarBookingId, companyId, userId, ",
-							"startTime, endTime from CalendarBooking where ",
-							"allDay = [$TRUE$]")));
+							"select calendarBookingId, companyId, ",
+							"ctCollectionId, userId, startTime, endTime from ",
```

Comment:
@rijema it should be `select ctCollectionId, calendarBookingId, companyId, userId, startTime, endTime` to follow the same order from [tables.sql](https://github.com/liferay/liferay-portal/blob/76041b96ce6b6cd7caf741a6da571a348a26801a/modules/apps/calendar/calendar-service/src/main/resources/META-INF/sql/tables.sql#L26-L44)